### PR TITLE
chore: replace pptr with pw

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: utils/testrunner/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@pptr/testrunner",
```
Comment (aslushnikov) [inline utils/testrunner/package.json]: let's remove package.json - we don't publish it atm

File: utils/testrunner/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This test runner is used internally by Playwright to test Playwright itself.
 ### Installation
 
 ```sh
-npm install --save-dev @pptr/testrunner
+npm install --save-dev @playwright/testrunner
```
Comment (aslushnikov) [inline utils/testrunner/README.md]: we can remove the installation part as well

File: utils/testrunner/README.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ node test.js
 ```
 
 ```js
-const {TestRunner, Reporter, Matchers} = require('@pptr/testrunner');
+const {TestRunner, Reporter, Matchers} = require('@playwright/testrunner');
```
Comment (aslushnikov) [inline utils/testrunner/README.md]: `require('.')`

File: utils/testserver/README.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ This test server is used internally by Playwright to test Playwright itself.
 ### Example
 
 ```js
-const {TestServer} = require('@pptr/testserver');
```
Comment (aslushnikov) [inline utils/testserver/README.md]: let's require `require('.')`

File: utils/testserver/package.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@pptr/testserver",
+  "name": "@playwright/testrunner",
```
Comment (aslushnikov) [inline utils/testserver/package.json]: let's drop package.json

**Review by aslushnikov — APPROVED** [review]

File: utils/testrunner/package.json (hunk 1 above)
Comment (yury-s) [reply utils/testrunner/package.json]: Done.

File: utils/testserver/package.json (hunk 5 above)
Comment (yury-s) [reply utils/testserver/package.json]: Done.

File: utils/testrunner/README.md (hunk 2 above)
Comment (yury-s) [reply utils/testrunner/README.md]: Done.

File: utils/testrunner/README.md (hunk 3 above)
Comment (yury-s) [reply utils/testrunner/README.md]: Done.

**Pull request merged**